FileMapper/getDriveIdAPI.js

```diff
@@ -0,0 +1,124 @@
+/**
```

Comment:
prefer file names to be of the form drive_id_helper.js (not camel case)

Also for all other files

Reply:
Okay.
Should i change it now, you mentioned to not change file names while PRs are being reviewed ?